src/Rector/Deprecation/Base/EntityViewBase.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace DrupalRector\Rector\Deprecation\Base;
+
+use DrupalRector\Utility\AddCommentTrait;
+use PhpParser\Node;
+use Rector\Core\Rector\AbstractRector;
+
+/**
+ * Replaced deprecated entity_load() calls.
```

Comment:
Can we combine `EntityViewBase.php` and `EntityViewRector`? It looks like there was a base class, which we could refactor later, but for now, putting them together makes more sense to me while reading. The comments seem like they need to be updated too.